markdown/extensions/tables.py

```diff
@@ -126,7 +126,7 @@ def _build_row(self, row, parent, align):
             except IndexError:  # pragma: no cover
                 c.text = ""
             if a:
-                c.set('align', a)
+                c.set('style', 'text-align:'+a)
```

Comment:
The only thing I would suggest is using a space in `style` attribute, and maybe adding a trailing semicolon.
```suggestion
                c.set('style', f'text-align: {a};')
```

Reply:
Yes, I agree with all three changes here. Add the space and the semicolon and use an f-string, which is the most performant way to combine strings.